docs/resources/cluster_v2.md

```diff
@@ -58,6 +58,44 @@ resource "rancher2_machine_config_v2" "foo" {
   }
 }
 
+# Create a new rancher v2 Cluster with multiple machine pools
+resource "rancher2_cluster_v2" "foo-rke2" {
+  name = "foo-rke2"
+  kubernetes_version = "v1.21.4+rke2r2"
+  enable_network_policy = false
+  default_cluster_role_for_project_members = "user"
+  rke_config {
+    machine_pools {
+      name = "pool1"
+      cloud_credential_secret_name = rancher2_cloud_credential.foo.id
+      control_plane_role = true
+      etcd_role = true
+      worker_role = false
+      quantity = 1
+      drain_before_delete = true
+      machine_config {
+        kind = rancher2_machine_config_v2.foo.kind
+        name = rancher2_machine_config_v2.foo.name
+      }
+    }
+    # Must be passed as individual each desired machine pool
```

Comment:
Update to `# Each machine pool must be passed separately`

Reply:
done